### PR TITLE
[COM-1410] Move Numbered and Bullet lists to main toolbar

### PR DESCRIPTION
**Pull request by guilean**
**Checklist**

- [x] New feature (non-breaking change which adds functionality)

**Please check if the PR fulfills these requirements**

- [x] Storybook or docs have been added / updated (for bug fixes / features)

**What is the ticket / issue associated with this change?**
https://impulsumvc.atlassian.net/browse/COM-1410

### CODE REVIEW

File: turbo.json [hunk 1]
```diff
@@ -29,11 +25,10 @@
     },
     "build-storybook": {
       "dependsOn": [
-        "@cmpsr/components#libbuild"
+        "@cmpsr/components#libbuild",
+        "@cmpsr/markdown-editor#libbuild"
```
Comment (tkanzakic) [inline turbo.json:29]: should we add blocks in here too?

File: packages/markdown-editor/src/MarkdownEditor/MarkdownEditor.stories.tsx [hunk 2]
```diff
@@ -5,10 +5,10 @@ import { EditorMode } from './types';
 
 export default {
   component: MarkdownEditor,
-  title: 'Components/Editors/MarkdownEditor',
+  title: 'Editors/MarkdownEditor',
```
Comment (tkanzakic) [inline packages/markdown-editor/src/MarkdownEditor/MarkdownEditor.stories.tsx:8]: why this change?

Comment (guilean) [reply packages/markdown-editor/src/MarkdownEditor/MarkdownEditor.stories.tsx]: Restored original.

**Review by tkanzakic — APPROVED** [review]
